GUIDELINES.md

```diff
@@ -0,0 +1,63 @@
+# Guidelines for the Tagging Schema
+
+Do you have an idea for a new preset or field? Read this!
+
+## 1. Evaluate Your Idea for the Tagging Schema Project
+
+Adding a preset or field to the tagging schema is a significant responsibility.
+We must ensure that both new and experienced users can understand the presets and fields,
+thereby contributing high-quality data to OpenStreetMap (OSM).
+
+Consider the following:
+
+### General Guidelines
+
+- 📋 **Established Documentation**: The tagging schema will only consider tags that are well-documented on the OSM wiki. The documentation should be clear and unambiguous.
+- 🏷️ **Established Tags Only**: No new or unestablished tags should be part of presets. Establishing tags must remain a community-driven process, not dictated by software implementation.
+- ✅ **Proposal or Accepted**: A tag is considered established when it has completed the [proposal process](https://wiki.openstreetmap.org/wiki/Proposal_process) or is otherwise accepted by the OSM community. Factors include the tag's duration of use, frequency of use, and whether its usage is increasing over time.
+- 🤷 **Useful Purpose**: Especially for less established tags, presets and fields should have a practical application. OSM allows for the collection of a wide variety of data, but not all of it is useful. For example, the brightness of street lamps might be documented, but it doesn't necessarily warrant a preset or field.
```

Comment:
One of these bullet points could mention support in mainstream data consumers as evidence that a tag is either established or useful. After all, this is one factor for classifying tags as _de facto_ on the wiki. (Mappers have occasionally been discouraged from starting a vote on a _de facto_ tag, due to the risk that a rejection might raise backwards compatibility concerns.)

Reply:
It now reads
> - ✅ **Proposal or Accepted**: A tag is considered established when it has completed the [proposal process](https://wiki.openstreetmap.org/wiki/Proposal_process) or is otherwise accepted by the OSM community. Factors include the tag's duration of use, frequency of use, and whether its usage is increasing over time.